### PR TITLE
tree, type: add Hindi translation

### PR DESCRIPTION
**Pull request by pranjalirathi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

File: pages.hi/windows/type.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# type
+
+> एक फाइल की सामग्रियों को प्रदर्शित कीजिए।
+> अधिक जानकारी: <https://learn.microsoft.com/windows-server/administration/windows-commands/type>.
+
+- किसी विशिष्ट फ़ाइल की सामग्री प्रदर्शित करें:
+
+`type {{पथ\से\फ़ाइल}}`
```
Comment (kbdharun) [inline pages.hi/windows/type.md:8]:
```suggestion
`type {{फ़ाइल\का\पथ}}`
```

File: pages.hi/windows/tree.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`tree {{पथ\से\निर्देशिका}}`
+
+- फ़ाइलों सहित निर्देशिका के लिए वृक्ष प्रदर्शित करें:
+
+`tree {{पथ\से\निर्देशिका}} /f`
+
+- विस्तारित वर्णों के बजाय ASCII वर्णों का उपयोग करके वृक्ष प्रदर्शित करें:
+
+`tree {{पथ\से\निर्देशिका}} /a`
```
Comment (kbdharun) [inline pages.hi/windows/tree.md:20]:
```suggestion
`tree {{निर्देशिका\का\पथ}}`

- फ़ाइलों सहित निर्देशिका के लिए वृक्ष प्रदर्शित करें:

`tree {{निर्देशिका\का\पथ}} /f`

- विस्तारित वर्णों के बजाय ASCII वर्णों का उपयोग करके वृक्ष प्रदर्शित करें:

`tree {{निर्देशिका\का\पथ}} /a`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have some suggestions for this page.

Note: We prefer having `निर्देशिका\का\पथ` (directory's path) instead of `पथ\से\निर्देशिका`  (path\to\directory) for Hindi translations as the later (which is kinda a transliteration to replicate the English format) isn't natural to read for native readers.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**